### DISCUSSION TITLE
New faction, sample configs from BI... ACE medical not working with hitpoints?

### DISCUSSION THREAD

**Discussion by IceBreakr-SBP** [acemod/ACE3#9173] [Q&A]
[ACE] (medical_engine) ERROR: Bad hitpoints for unit type "XY".
 I've tried almost everything with my units... they are based on samples given out by BI.
 
 Can someone assist me with this? What am I missing here?

**Comment by Dystopian** [comment]
@IceBreakr-SBP  I'm not good in ACE medical but I see this error here: https://github.com/acemod/ACE3/blob/274d853c433029e67624ba598c6c72266d7f84d4/addons/medical_engine/XEH_postInit.sqf#L18-L22
Also it can be checked with this dev script: https://github.com/acemod/ACE3/blob/274d853c433029e67624ba598c6c72266d7f84d4/addons/medical/dev/test_hitpointConfigs.sqf#L3

Both scripts check if the **last** unit HitPoint is `ACE_HDBracket`. It's added with this ACE macro https://github.com/acemod/ACE3/blob/274d853c433029e67624ba598c6c72266d7f84d4/addons/medical_engine/script_macros_config.hpp#L51
in this config https://github.com/acemod/ACE3/blob/274d853c433029e67624ba598c6c72266d7f84d4/addons/medical_engine/CfgVehicles.hpp#L4-L10

It's added for `CAManBase` and some more specific unit classes like `B_Soldier_04_f` or `I_E_Uniform_01_coveralls_F` (I don't really know why, maybe inheritance bug).

Maybe your units have rewritten HitPoint config and `ACE_HDBracket` is missing or not **last** one which is critical for ACE medical.

Hope it helps.
